src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
+  const ignorePhrases = [
+    "@presubmit ignore",
+    "@presubmit: ignore",
+    "@presubmit skip",
+    "@presubmit: skip",
+    "@presubmitai ignore",
+    "@presubmitai: ignore",
+    "@presubmitai skip",
+    "@presubmitai: skip",
+  ];
+  const bodyLower = (pull_request.body ?? "").toLowerCase();
+
+  for (const phrase of ignorePhrases) {
+    if (bodyLower.includes(phrase.toLowerCase())) {
```

Comment:
The `toLowerCase()` is being called on each `phrase` inside the loop, even though the phrases are constant strings. Consider converting the `ignorePhrases` array to lowercase when it's defined to avoid redundant string transformations:

```typescript
const ignorePhrases = [
  '@presubmit ignore',
  // ... other phrases
].map(phrase => phrase.toLowerCase());

// Then in the loop
if (bodyLower.includes(phrase)) {
```

This way, the lowercase conversion happens only once per phrase instead of on every iteration.